build/linux/dist/install.sh

```diff
@@ -205,6 +205,14 @@ simple_uninstall_f() {
 # Update desktop file and mime databases (if possible)
 updatedbs_f() {
 
+  if [ if "${HOME}/root/.config/mimeapps.list"]
+    then
+      continue
+  else
```

Comment:
These lines are unnecessary if you invert your logic using the NOT Boolean operator `!` (`if [ ! ...`)